src/test/mochitest/head.js

```diff
@@ -223,22 +228,17 @@ function waitForSelectedSource(dbg) {
  * @param {Number} line
  * @static
  */
-function assertPausedLocation(dbg, source, line) {
+function assertPausedLocation(dbg) {
```

Comment:
we're dropping specified source and lines because we want to focus on the fact that we're highlighting the top frame line.